src/components/connection_handler/test/connection_handler_impl_test.cc

```diff
@@ -1172,13 +1191,13 @@ TEST_F(ConnectionHandlerTest, SessionStop_CheckHash) {
 
     const uint32_t end_audio_wrong_hash =
         connection_handler_->OnSessionEndedCallback(
-            uid_, start_session_id_, &wrong_hash, kRpc);
+            uid_, out_context_.new_session_id_, &wrong_hash, kRpc);
     EXPECT_EQ(0u, end_audio_wrong_hash);
     EXPECT_EQ(protocol_handler::HASH_ID_WRONG, wrong_hash);
-    CheckSessionExists(uid_, start_session_id_);
+    CheckSessionExists(uid_, out_context_.new_session_id_);
 
     const uint32_t end_audio = connection_handler_->OnSessionEndedCallback(
-        uid_, start_session_id_, &hash, kRpc);
+        uid_, out_context_.new_session_id_, hash, kRpc);
```

Comment:
`&` was deleted here, this means that the deprecated version of the function is being used. Use `&hash`

Reply:
@jacobkeeler fixed in https://github.com/smartdevicelink/sdl_core/pull/1971/commits/c95042bdce0bc3d598ce8300e7b17d7d55e9e4b4